README.md

```diff
@@ -54,7 +54,7 @@ The .NET Core configuration system is used to determine the state of feature fla
 
 The feature management library supports appsettings.json as a feature flag source since it is a provider for .NET Core's IConfiguration system. Below we have an example of the format used to set up feature flags in a json file.
 
-``` JavaScript
+``` json

```

Comment:
```suggestion
``` JavaScript
```
This should stay JavaScript. Its a trick to allow "//" comments without github highlighting it in red. (See rich diff)

Reply:
Oh, thanks. I never noticed this difference.